README.md

```diff
@@ -25,6 +25,8 @@
 * [Apache Tomcat](http://tomcat.apache.org/)
 * [Debugger for Java](https://marketplace.visualstudio.com/items?itemName=vscjava.vscode-java-debug) (for debuging war package)
 
+If you are a Windows user, you should add the bin subdirectory of your JDK installation to your user-level or system PATH environment variable.
```

Comment:
you should add the bin sub-directory of your JDK installation to make sure the 'java' command is available in your user-level or system PATH environment variable